### PR TITLE
Gift

### PR DESCRIPTION
**Pull request by LuisCastilloKC**
Add the CRUD for Gift with just one Attribute. I will add more attributes as we continue.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,4 +3,6 @@
   post "/login", to: "auth#login"
   get "/auto_login", to: "auth#auto_login"
   get "/user_is_authed", to: "auth#user_is_authed"
+
+  resources :gits
```
Comment (4nth0ny1) [inline config/routes.rb]: should be gifts, right?

File: app/controllers/gifts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class GiftsController < ApplicationController
+
+    def index
+        gift = Gift.all
+        render json: gift
+    end
+
```
Comment (4nth0ny1) [inline app/controllers/gifts_controller.rb:7]: should this be gifts because we are grabbing all of them?

**Review by 4nth0ny1 — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (LuisCastilloKC) [reply config/routes.rb]: Yes, you are correct. It is a typo. It should be fixed now

File: app/controllers/gifts_controller.rb (hunk 2 above)
Comment (LuisCastilloKC) [reply app/controllers/gifts_controller.rb:7]: Yes, it should be plural. It should be fixed now.

**Review by 4nth0ny1 — APPROVED** [review]

**Pull request merged**